template_app_ipfire_by_zabbix_agent_active.xml

```diff
@@ -791,6 +794,48 @@ Created by Robin Roevens (robin.roevens (at) disroot.org)</description>
                 </macro>
             </macros>
         </template>
+        <template>
+            <uuid>85fea82fd2be4f5fb7e0ba93e268f5a6</uuid>
+            <template>Template Module IPFire DHCP Clients by Zabbix Agent Active</template>
+            <name>Template Module IPFire DHCP Clients by Zabbix Agent Active</name>
+            <description>Monitor DHCP clients of IPFire Services
+Created by Stack Korora ( stackkorora (at) disroot.org )</description>
+            <groups>
+                <group>
+                    <name>Templates/Modules</name>
+                </group>
+            </groups>
+            <items>
+                <item>
+                    <uuid>437fe278f4984f42917039aff8fd6c28</uuid>
+                    <name>IPFire Captive Portal Clients</name>
+                    <type>ZABBIX_ACTIVE</type>
+                    <key>ipfire.captive.clients</key>
+                    <history>1d</history>
+                    <description>Number of active users on captive portal</description>
+                    <tags>
+                        <tag>
+                            <tag>Application</tag>
+                            <value>IPFire Captive Portal</value>
+                        </tag>
+                    </tags>
+                </item>
+                <item>
+                    <uuid>0e8d3a7d6c1a4224b388bdcbb4c31eb7</uuid>
+                    <name>IPFire DHCPd Leases</name>
+                    <type>ZABBIX_ACTIVE</type>
+                    <key>ipfire.dhcp.clients</key>
+                    <history>1d</history>
+                    <description>Number of active dhcpd leases on captive portal</description>
```

Comment:
This description should not mention "on captive portal" as it has nothing to do with the captive portal an sich.
```suggestion
                    <description>Number of active dhcpd leases</description>
```

Reply:
Great catch. Fully agree.